lib/salus/config.rb

```diff
@@ -48,7 +50,14 @@ def initialize(configuration_files = [], ignore_ids = [])
       # defined in the earlier files in the array (DEFAULT_CONFIG is the first such file/object).
       final_config = DEFAULT_CONFIG.dup
       configuration_files.each do |file|
-        filtered_data = filter_ignored_ids(YAML.safe_load(file), ignore_ids)
+        file_content = YAML.safe_load(file)
+        if file_content.is_a?(Hash)
+          filtered_data = filter_ignored_ids(file_content, ignore_ids)
+        else
+          msg = "file_content #{file_content.inspect} is not a hash"
+          bugsnag_notify(msg)
+          filtered_data = {}
```

Comment:
We will cover a few additional bases if you move filtered_data = {} to line before the YAML_safe_load.  That will help if the safe_load runs into any classes not whitelisted (Disallowed classes).  Probably not a big deal not but would future proof the code here.

Rest looks good!